frontend/src/components/DropDown/index.jsx

```diff
@@ -1,20 +1,98 @@
+  { label: "History", value: "History" }, { label: "Indonesian", value: "Indonesian" },
+  { label: "Iranian", value: "Iranian" }, { label: "Italian", value: "Italian" },
+  { label: "Japanese", value: "Japanese" }, { label: "Jewish Studies", value: "Jewish Studies" },
+  { label: "Korean", value: "Korean" }, { label: "Lesbian, Gay, Bisexual, Transgender, and Queer Studies", value: "Lesbian, Gay, Bisexual, Transgender, and Queer Studies" },
+  { label: "Latin", value: "Latin" }, { label: "Linguistics", value: "Linguistics" },
+  { label: "Law", value: "Law" }, { label: "Management", value: "Management" },
+  { label: "Medicine", value: "Medicine" }, { label: "Management-Executive MBA", value: "Management-Executive MBA" },
+  { label: "Microbiology, Immunology, and Molecular Genetics", value: "Microbiology, Immunology, and Molecular Genetics" }, 
+  { label: "Management-Full-Time MBA", value: "Management-Full-Time MBA" },
+  { label: "Management-Fully Employed MBA", value: "Management-Fully Employed MBA" }, { label: "Molecular, Cellular, and Integrative Physiology", value: "Molecular, Cellular, and Integrative Physiology" },
+  { label: "Mathematics", value: "Mathematics" }, { label: "Music", value: "Music" },
+  { label: "Musicology", value: "Musicology" }, { label: "Nursing", value: "Nursing" },
+  { label: "Pathology and Laboratory Medicine", value: "Pathology and Laboratory Medicine" }, 
+  { label: "Philosophy", value: "Philosophy" },
+  { label: "Physics", value: "Physics" }, { label: "Physics and Biology in Medicine", value: "Physics and Biology in Medicine" },
+  { label: "Psychology", value: "Psychology" }, { label: "Physiological Science", value: "Physiological Science" },
+  { label: "Polish", value: "Polish" }, { label: "Religion, Study of", value: "Religion, Study of" },
+  { label: "Russian", value: "Russian" }, { label: "Scandinavian", value: "Scandinavian" },
+  { label: "Sociology", value: "Sociology" }, { label: "Semitic", value: "Semitic" },
+  { label: "Slavic", value: "Slavic" }, { label: "Spanish", value: "Spanish" },
+  { label: "Social Science", value: "Social Science" }, { label: "Statistics", value: "Statistics" },
+  { label: "Swahili", value: "Swahili" }, { label: "Thai", value: "Thai" },
+  { label: "Turkic Languages", value: "Turkic Languages" }, { label: "Theater", value: "Theater" },
+  { label: "Yiddish", value: "Yiddish" },
 ];
 
-function AppDropDown({ onSelect }) {
+const AppDropDown = () => {
+
+  // Test with some dummy values if selected subject = Art (dummy values)  
+  // const entries = ["Art_1A", "Art_1B", "Art_11A", "Art_11B"];
+
+  // // Testing purpose 
+  // const handleGetClassList2 = () => {
+  //   let subclassFromApi = entries.map((item) => {
+  //     return {label: item, value: item};
+  //   });
+
+  //   setClassList(subclassFromApi);
+  // }
+  // --------------------------------
+
+  const [classList, setClassList] = useState([]);
+  const [selectedClass, setSelectedClass] = useState("");
```

Comment:
It would be better if we could put selectedClass / setSelectedClass as passed-in params so that it could be used outside.

Reply:
Where should I put the selectedClass / setSelectedClass for outside? 
Are you saying that if i want to use them, I would pass in as parameters from outside?